app/controllers/observation_quality_experiments_controller.rb

```diff
@@ -0,0 +1,101 @@
+# frozen_string_literal: true
+
+class ObservationAccuracyExperimentsController < ApplicationController
```

Comment:
This file should be named `observation_accuracy_experiments_controller.rb`, not `quality`